bifrost-fastify/index.ts

```diff
@@ -85,6 +85,12 @@ export const viteProxyPlugin: FastifyPluginAsync<
     upstream: upstream.href,
     async preHandler(req, reply) {
       if (req.method === "GET" && req.accepts().type(["html"]) === "html") {
+        if (req.url.endsWith('/')) {
```

Comment:
is `.url` the whole url or just the path? SHould have test that this handles query params etc.

Reply:
good point, modified to handle this